lib/matplotlib/backends/backend_wx.py

```diff
@@ -1406,13 +1406,13 @@ def pan(self, *args):
         self.ToggleTool(self.wx_ids['Zoom'], False)
         NavigationToolbar2.pan(self, *args)
 
-    def configure_subplots(self, evt):
```

Comment:
very technically this is also a backwards incompatible change because one cannot pass `evt` as kwarg anymore, but I'm not going to lose sleep over that (the signature is now consistent with `pan` and `zoom`).